src/rendering/component/title.rs

```diff
@@ -49,32 +49,32 @@ pub(in crate::rendering) fn render<B: Backend>(
         [width - PADDING, height + TEXT_ALIGN_BOTTOM],
         DEFAULT_TEXT_SIZE,
         [text_color; 2],
-    );
+    ) - PADDING;
 
     let (line1_y, line1_end_x) = if !component.line2.is_empty() {
         let line2 = context.choose_abbreviation(
             component.line2.iter().map(|a| &**a),
             DEFAULT_TEXT_SIZE,
-            line2_end_x - PADDING - left_bound,
+            line2_end_x - left_bound,
         );
         context.render_text_align(
             line2,
             left_bound,
-            line2_end_x - PADDING,
+            line2_end_x,
             [line_x, height + TEXT_ALIGN_BOTTOM],
             DEFAULT_TEXT_SIZE,
             component.is_centered,
             text_color,
         );
         (TEXT_ALIGN_TOP, width - PADDING)
     } else {
-        (height / 2.0 + TEXT_ALIGN_CENTER, line2_end_x - PADDING)
+        (height / 2.0 + TEXT_ALIGN_CENTER, line2_end_x)
     };
 
     let line1 = context.choose_abbreviation(
         component.line1.iter().map(|a| &**a),
         DEFAULT_TEXT_SIZE,
-        width - PADDING - left_bound,
+        line1_end_x - left_bound,
```

Comment:
For choosing the abbreviation here, we accidentally took the right border of the title component instead of taking into account that there might only be a single line and us sharing the line with the attempt count then. In that case the abbreviation needs to be chosen in respect to how much space it takes up (which is `line1_end_x`).